scripts/ansible/roles/windows/az-dcap-client/vars/windows.yml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+---
+psw_archive_url: 'http://registrationcenter-download.intel.com/akdlm/irc_nas/15018/Intel%20SGX%20PSW%20for%20Windows%20v2.2.100.48339.exe'
+dcap_archive_url: 'http://registrationcenter-download.intel.com/akdlm/irc_nas/15017/Intel%20SGX%20DCAP%20for%20Windows%20v1.0.100.48134.exe'
```

Comment:
Intel has since pushed out a newer version of the package at: http://registrationcenter-download.intel.com/akdlm/irc_nas/15025/Intel%20SGX%20DCAP%20for%20Windows%20v1.0.101.48697.exe

Reply:
I'll update this.

Reply:
@ionutbalutoiu - It appears that you haven't updated to this version of the package.